common-healing.lic

```diff
@@ -174,15 +174,18 @@ module DRCH
       $severity_to_text.each do |severity_level, match_text|
         next unless wound =~ Regexp.union(match_text)
         match_names = Regexp.last_match.names
-        part = if match_names.include?('skin')
-                 'skin'
-               elsif match_names.include?('abdomen')
-                 'abdomen'
-               elsif match_names.include?('head')
-                 'head'
-               else
-                 Regexp.last_match(:part)
-               end
+        if match_names.include?('skin')
+          part = 'skin' if !Regexp.last_match[:skin].nil?
+        end
+        if match_names.include?('abdomen')
+          part = 'abdomen' if !Regexp.last_match[:abdomen].nil?
+        end
+        if match_names.include?('head')
+          part = 'head' if !Regexp.last_match[:head].nil?
+        end
+        if match_names.include?('part')
+          part = Regexp.last_match(:part) if !Regexp.last_match[:part].nil?
+        end
```

Comment:
I would love to know if there is a cleaner way to do this assignment to `part`. In the previous implementation, the capture group names are present in `last_match` even if they don't match anything. I misunderstood how it worked. I thought capture group names would only show up if they captured anything.